insolar/message/bootstrap.go

```diff
@@ -21,19 +21,22 @@ import (
 )
 
 // GenesisRequest is used for genesis records generation.
+// this is fake message that never passed to messageBus
+// it implements Message Interface for ability to be converted
+// to Parcel and than be passed to RegisterRequest method
 type GenesisRequest struct {
```

Comment:
Rename also TypeBootstrapRequest to TypeGenesisRequest and update its comment too

Reply:
done